skimage/restoration/tests/test_denoise.py

```diff
@@ -35,22 +36,24 @@
 astro_odd = astro[:, :-1]
 
 
-def test_denoise_tv_chambolle_2d():
+@testing.parametrize('dtype', [np.float32, np.float64])
```

Comment:
Actual question; Why are we limiting ourselves to the two float types here? Wouldn't it be a more thorough test to check for all the types that `_float_type` cares about?

Reply:
Sure, I think we can and float16 and float128 here to make sure they are also handled as expected.